config/vufind/EDS.ini

```diff
@@ -335,6 +335,15 @@ AU = None
 ; This repeatable setting can be used to filter out values using group codes:
 ;excludeGroup[] = 'URL'
 
+; This section can be used to order items based on their labels.
```

Comment:
It may be worth mentioning that any items defined here will appear before any items not defined here.  I.e. if the only value supplied is `5 = Title`, the title will still be listed first.

Reply:
Actually I see my statement above was wrong, it's more complicated than that.  If the only value supplied is `500 = Title`, then title will actually appear last.  Since for 'n' original items the defaults are ordered between 'n' and '2n'.  So perhaps easier just to tell people they should start the order at 1, and not to tell them what happens if they don't. :)

Reply:
Yeah agreed! I'll write a patch doing that

Reply:
I've pushed this change in the follow-up patchset :)